src/app/page.tsx

```diff
@@ -27,28 +25,11 @@ export default function Home() {
           <p>คำถามที่พบบ่อย</p>
         </div>
         <div className="flex flex-col items-center gap-6">
-          {MOCK_FAQ.map((item, index) => (
-            <div key={index} className="w-full">
-              {item.title.length > 0 && (
-                <SectionHeader title={item.title} className="mb-4" />
-              )}
-              <LayoutGroup>
-                <Accordion
-                  type="multiple"
-                  className="flex w-full flex-col items-center gap-3 px-4"
-                >
-                  {item.questions.map((q, qIndex) => (
-                    <FaqCard
-                      key={qIndex}
-                      question={q.question}
-                      answer={q.answer}
-                      value={`faq-${index}-${qIndex}`} //unique value
-                    />
-                  ))}
-                </Accordion>
-              </LayoutGroup>
-            </div>
-          ))}
+          <LayoutGroup>
+            {MOCK_FAQ.map((q, index) => (
+              <Accordion key={index} questions={q.questions} title={q.title} />
+            ))}
```

Comment:
Here you’ll see the fruits of the refactor I suggested. This code block would just be:

```tsx
{MOCK_FAQ.map((section) => (
  <Accordion key={section.id} section={section} />
))}
```